templates/huginn/0/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:9.5
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: huginn
+      POSTGRES_DB: huginn
```

Comment:
I would add catalog questions for DB_USER and DB_NAME. 
```
-      POSTGRES_USER: huginn
-      POSTGRES_DB: huginn
+      POSTGRES_USER: ${DB_USER}
+      POSTGRES_DB: ${DB_NAME}
```

Reply:
I can do that, but may I ask why you would want to do that?  :question: Just curious. :slightly_smiling_face: